libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransition.kt

```diff
@@ -267,12 +280,22 @@ class MapboxNavigationCameraTransition(
                     owner(NAVIGATION_CAMERA_OWNER)
                 }
             ) {
-                duration = LINEAR_ANIMATION_DURATION
-                interpolator = LINEAR_INTERPOLATOR
+                startDelay = pitchAndPaddingDelay.toLong()
+                duration = pitchAndPaddingDuration.toLong()
+                interpolator = SLOW_OUT_SLOW_IN_INTERPOLATOR
             }
             animators.add(paddingAnimator)
         }
 
         return createAnimatorSet(animators).constraintDurationTo(transitionOptions.maxDuration)
     }
+
+    private fun flyFromLowZoomToHighZoom(
+        cameraOptions: CameraOptions,
+        pluginImpl: CameraAnimationsPluginImpl,
+        transitionOptions: NavigationCameraTransitionOptions
+    ): AnimatorSet {
+        val animators = pluginImpl.cameraAnimationsFactory.getFlyTo(cameraOptions)
+        return createAnimatorSetWith(animators).setDuration(transitionOptions.maxDuration)
```

Comment:
This duration has to be variable, otherwise, cross-globe transitions look rather fast but the very short transitions are extremely slow (if the camera is already next to the puck). Maybe we can try re-using the duration calculation from the previous function.

Reply:
Please take a look at the change. I am calculating the animation duration for `center`. However, based on how far are you from your current location, the pixel distance can range between 1-4 digits (ex: screenDistance = 0 - 1500). The smaller the distance, smaller would be the duration for the center animator. Hence, I have forced a minimum duration for center animator `coerceAtLeast(1000L)` and maximum duration to be `coerceAtMost(3000L)`. I am setting this as the duration to `cameraAnimatorSet` and constranining it to a maximum of `transitionOptions.maxDuration`. This looks good to me. Let me know what do you think about it
@LukasPaczos